### ISSUE TITLE
Implement some unit testing with Jest

### ISSUE THREAD

**Issue by chris-steele** [adaptlearning/adapt-contrib-core#236]
Demo setup of a unit test using Jest that allows the plugin to be tested in isolation.

**Comment by chris-steele** [comment]
Murmurings of unit testing was mentioned so keen to hear what you think @eleanor-heath and @oliverfoster

**Comment by eleanor-heath** [comment]
Hi @chris-steele, Ive added a comment to your commit [here](https://github.com/adaptlearning/adapt-contrib-core/commit/2649a6d44f4ba5079caa3fc87644df861cffbdfc#commitcomment-85903604)

**Comment by chris-steele** [comment]
Hi @eleanor-heath and @oliverfoster Just an update on this. The unit tests in this example relate to `filterIntersectingHierarchy` which has been moved to [adapt-contrib-scoring](https://github.com/cgkineo/adapt-contrib-scoring). Accordingly, I've set up the example in [this fork](https://github.com/chris-steele/adapt-contrib-scoring). Also, I've set up a GitHub Action to run the tests when a PR is created. Have a look at the example: https://github.com/chris-steele/adapt-contrib-scoring/pull/2. If you click 'Show all checks' and then 'Details' you'll see the output of the test runner.
